File: gems/decomplex/examples/typescript/sequence-mine.ts
function one() { alloc_mark(x); body1(); cleanup(x); }
function two() { alloc_mark(y); body2(); cleanup(y); }
function three() { alloc_mark(z); body3(); cleanup(z); }
function four() { alloc_mark(w); body4(); cleanup(w); }
function leak() { alloc_mark(q); use_value(q); }

File: gems/decomplex/examples/c/sequence-mine.c
void one() { alloc_mark(x); body1(); cleanup(x); }
void two() { alloc_mark(y); body2(); cleanup(y); }
void three() { alloc_mark(z); body3(); cleanup(z); }
void four() { alloc_mark(w); body4(); cleanup(w); }
void leak() { alloc_mark(q); use_value(q); }
